.github/workflows/release-check.yml

```diff
@@ -8,4 +8,4 @@ on:
 
 jobs:
   release-check:
-    uses: protocol/.github/.github/workflows/release-check.yml@master
+    uses: protocol/.github/.github/workflows/release-check.yml@fork-release-check
```

Comment:
not sure about the implications of this... @marten-seemann ?

Reply:
I'm just investigating https://github.com/protocol/.github/issues/254 here. I'll close this PR once I'm done. 

This change you tagged here means that the `release-check` under this PR https://github.com/libp2p/go-doh-resolver/runs/4559777662?check_suite_focus=true used the action defined here https://github.com/protocol/.github/blob/fork-release-check/.github/workflows/release-check.yml instead of the one from `master`.